src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,15 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
 ICD10WHO,https://icd.who.int/browse10/2019/en#/
+ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
 OMIMPS,https://www.omim.org/phenotypicSeries/PS
 OMIM,https://omim.org/entry/
 Orphanet,http://www.orpha.net/ORDO/Orphanet_
-GARD,http://purl.obolibrary.org/obo/GARD_
\ No newline at end of file
+GARD,http://purl.obolibrary.org/obo/GARD_
+MEDRA,http://identifiers.org/meddra/
```

Comment:
## ~TODO?: `mondo.sssom.config.yml` @joeflack4 
<details><summary>Details</summary>
<p>

I realize now that this is actually not consistent with `prefixes.csv`, but this is somewhat moot and will be made redundant by:
- #146

</p>
</details>